src/Client.php

```diff
@@ -354,4 +365,47 @@ public function watch(array $pipeline = [], array $options = [])
 
         return $operation->execute($server);
     }
+
+    private static function getVersion() : string
+    {
+        if (self::$version === null) {
+            try {
+                self::$version = PrettyVersions::getVersion('mongodb/mongodb')->getPrettyVersion();
+            } catch (OutOfBoundsException $e) {
+                return 'unknown';
```

Comment:
I took a look through PrettyVersions and it seems that it doesn't throw anything on its own, so I assume this comes from Ocramius' library. If so, I think there's also a case where [UnexpectedValueException could be thrown](https://github.com/Ocramius/PackageVersions/blob/651c372efc914aea8223e049f85afaf65e09ba23/src/PackageVersions/Versions.php#L27). Would it be safer to just catch any RuntimeException?

Also, I'm totally on board with _not_ exposing the [original exception](https://github.com/Ocramius/PackageVersions/blob/651c372efc914aea8223e049f85afaf65e09ba23/src/PackageVersions/FallbackVersions.php#L95). I don't think "fix your shit" would go over well with users, especially when it might not be clear which library is generating that message :laughing:

Reply:
Changed to catch any throwable. That exception message (along with not wanting to break a connection to MongoDB because we couldn't determine the version) was the main reason for adding a try-catch block.